### DISCUSSION TITLE
Internal tables are not loading

### DISCUSSION THREAD

**Discussion by GuilhermeHirsch** [Budibase/budibase#2186] [Help]
A strange behavior started happening in my app. The two tables I have are loading, but they don't finish. These are BB internal tables. Since I have this app working very well for days. I cannot lose this data.

https://www.loom.com/share/e28053d1f6c849b8b96264f18fc414df

**Comment by mike12345567** [comment]
Hi @GuilhermeHirsch - are there any errors occurring in the console/network tab of the developer tools, or is the network simply timing out?

**Reply by GuilhermeHirsch** [reply]
Hi, @mike12345567!!!

![Sem título](https://user-images.githubusercontent.com/33108596/127565318-d47acb93-1880-46ac-a75f-fd35366e43ae.jpg)

**Reply by mike12345567** [reply]
Hi @GuilhermeHirsch - looks like somethings gone wrong there, don't worry the data will still be in the database, I just expect there is some incompatibility thats stopping them from being retrieving right now.

Would it be possible for you to find the actual error the network call is giving back? Below I've posted a picture of accessing the network tab and previewing the response, you should see the rows requests that are failing appearing in read.
![image](https://user-images.githubusercontent.com/4407001/127565548-949506cc-32cb-46fc-b5da-96fc84bcefce.png)

**Reply by GuilhermeHirsch** [reply]
This data?

![Sem título1](https://user-images.githubusercontent.com/33108596/127566308-8ea43bce-2a2d-4fec-9589-dc083e6ff407.jpg)

**Reply by mike12345567** [reply]
Hi @GuilhermeHirsch - sorry if you click on one of those rows API calls you'll get a screen opens, which lists more information and you can click into "preview" which shows what the server responded with.

**Reply by GuilhermeHirsch** [reply]
![Sem título1](https://user-images.githubusercontent.com/33108596/127566656-34b17e82-3abd-48e0-8ace-0688968ff5ec.jpg)

**Reply by mike12345567** [reply]
Hi @GuilhermeHirsch - thanks for this, I will try to re-create with this info and get a fix out.

If possible, could you confirm if it still happens after exporting and importing it as a new app? If this is the case, I could likely solve your issue more quickly if you could privately share an example of an app which is exhibiting this problem with me.

**Reply by GuilhermeHirsch** [reply]
Table data was loaded when imported into a new app.

**Reply by mike12345567** [reply]
Interesting, did you export the table data or did you export and re-import the app itself?

**Reply by GuilhermeHirsch** [reply]
Export and re-import the app.
When I re-imported the app. The "home" screen did not load as before.

I sent the file to your email.

**Reply by mike12345567** [reply]
Thanks @GuilhermeHirsch I will try to replicate your issues and get them fixed, thanks for your help!

**Reply by GuilhermeHirsch** [reply]
I appreciate your help!
